portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsSingleSelectionField.testcase

```diff
@@ -153,13 +153,58 @@ definition {
 	}
 
 	@description = "Verify that options can be added"
-	@ignore = "Test Stub"
 	@priority = "5"
 	test OptionsCanBeAdded {
 		property forms.functional = "true";
 		property portal.acceptance = "false";
 
-		// TODO LPS-132205 OptionsCanBeAdded pending implementation
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			portlet = "Forms");
+
+		FormsAdminNavigator.gotoAddForm();
+
+		Form.editName(formName = "Form 1");
+
+		Form.gotoAddField(
+			fieldPositionNumber = "0",
+			fieldType = "Single Selection");
+
+		Form.save();
+
+		Click(
+				key_fieldName = "Single Selection",
+				locator1 = "Form#FIELD_LABEL");
+
+		Type(
+			key_rowNumber = "1",
+			locator1 = "FormFields#OPTIONS_OPTION_VALUE_FIELD",
+			value1 = "Option 1");
+
+		Type(
+			key_rowNumber = "2",
+			locator1 = "FormFields#OPTIONS_OPTION_VALUE_FIELD",
+			value1 = "Option 2");
```

Comment:
Hi @PauloVAS ,
I'd like to ask you to use FormFields.editOptionsRow to edit a Single Selection option.
This macro is going to allow you to edit this type of field while passing its field name as a parameter.

In this scenario, you would use something like this to edit the first option:

```
FormFields.editOptionsRow(			
			optionName = "Option1",			
			optionValue = "Option 1");
```
			
You can find an example [here](https://github.com/liferay/liferay-portal/blob/fdcbefc43088ad9bb8a05e51c9b5d22dbf507589/portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/Forms.testcase#L4594-L4597).